data/service/service/standard.go

```diff
@@ -273,3 +276,30 @@ func (s *Standard) initializeServer() error {
 
 	return nil
 }
+
+func getenvStr(key string) (string, error) {
```

Comment:
isn't there a similar function in this huge project already implemented?

Reply:
did not find any  but maybe i missed in my search